### PR TITLE
[ENG-1911] Allow buying multiple items

### PR DESCRIPTION
**Pull request by maximgeerinck**
Allow buying multiple items

### CODE REVIEW

File: src/offer/MintOffer.ts [hunk 1]
```diff
@@ -212,12 +212,23 @@ export class MintOffer extends BasicOffer {
...
     const contract = await this.getContract();
 
+    const price = this._chain.parseCurrency(
+      this._offer.price.currency,
+      this._offer.price.amount
+    );
+
+    const priceWithServiceFee = await this._chain.getPriceWithBuyServiceFee(
+      this._offer.price,
+      this._offer.marketConfig.buyServiceFeeBps.value,
```
Comment (sneljo1) [inline src/offer/MintOffer.ts:222]: Primary sale does not have buyer fee

File: src/refinable/RefinableBaseClient.ts [hunk 2]
```diff
@@ -142,10 +140,8 @@ export abstract class RefinableBaseClient<O extends object = {}> {
     id: string,
     storeId?: string
   ): Promise<O> {
-    const queryResponse = await this.apiClient.request<
-      GetOfferQuery,
-      GetOfferQueryVariables
-    >(GET_OFFER, {
+    // can not be typed as we use `parse` and it's not supported by typegen
+    const queryResponse = await this.apiClient.request(GET_OFFER, {
```
Comment (sneljo1) [inline src/refinable/RefinableBaseClient.ts:144]: This can be typed. You just have to add the `gql` keyword before the `. Worked for me

**Review by sneljo1 — CHANGES_REQUESTED** [review]

File: src/refinable/RefinableBaseClient.ts (hunk 2 above)
Comment (maximgeerinck) [reply src/refinable/RefinableBaseClient.ts]: added

File: src/offer/MintOffer.ts (hunk 1 above)
Comment (maximgeerinck) [reply src/offer/MintOffer.ts]: changed

**Review by sneljo1 — APPROVED** [review]

**Pull request merged**